### PR TITLE
Fix style and typo in document

### PR DESCRIPTION
**Pull request by okuta**

### CODE REVIEW

File: chainer/functions/array/broadcast.py [hunk 1]
```diff
@@ -54,11 +54,11 @@ def broadcast(*args):
     """Broadcast given variables.
 
     Args:
-      args (Variables): Variables to be broadcasted.
+        args (Variables): Variables to be broadcasted.
 
     Returns:
-      ``tuple``: Tuple of :class:`~chainer.Variable` objects which are
-          broadcasted from given arguments.
+        tuple: Tuple of :class:`~chainer.Variable` objects which are
```
Comment (unnonouno) [inline chainer/functions/array/broadcast.py]: `tuple` refers a method which makes a tuple object, not a type itself. Is that OK?
https://docs.python.org/2/library/functions.html#tuple

File: chainer/functions/loss/negative_sampling.py [hunk 2]
```diff
@@ -170,17 +170,17 @@ def negative_sampling(x, t, W, sampler, sample_size):
...
-    where :math:`\sigma(\cdot)` is a sigmoid function, :math:`w_i` is the
+    where :math:`\\sigma(\\cdot)` is a sigmoid function, :math:`w_i` is the
     weight vector for the word :math:`i`, and :math:`p` is a positive example.
     It is approximeted with :math:`k` examples :math:`N` sampled from
     probability :math:`P(i)`, like this:
 
     .. math::
 
-       f(x, p) \\approx \\log \\sigma(x^\\top w_p) + \\
+       f(x, p) \\approx \\log \\sigma(x^\\top w_p) +  \\\\
```
Comment (unnonouno) [inline chainer/functions/loss/negative_sampling.py]: Does a new line requires? Four backslashes makes two backslashes in TeX that represents a newline in TeX.

File: chainer/functions/math/matmul.py [hunk 3]
```diff
@@ -259,12 +259,12 @@ def batch_matmul(a, b, transa=False, transb=False):
 
     Args:
         a (Variable): The left operand of the batch matrix multiplications.
-            A 2-D array of shape (B, N,) is considered as B Nx1 matrices.
-            A 3-D array of shape (B, M, N) is considered as B MxN matrices.
+            A 2-D array of shape ``(B, N,)`` is considered as B Nx1 matrices.
```
Comment (unnonouno) [inline chainer/functions/math/matmul.py]: `,` is redundant?

File: chainer/functions/pooling/average_pooling_2d.py [hunk 4]
```diff
@@ -123,7 +123,7 @@ def average_pooling_2d(x, ksize, stride=None, pad=0, use_cudnn=True):
             specified, then it uses same stride as the pooling window size.
         pad (int or pair of ints): Spatial padding width for the input array.
             ``pad=p`` and ``pad=(p, p)`` are equivalent.
-        use_cudnn (bool): If True and CuDNN is enabled, then this function
+        use_cudnn (bool): If ``True`` and CuDNN is enabled, then this function
```
Comment (unnonouno) [inline chainer/functions/pooling/average_pooling_2d.py]: cuDNN

**Comment by unnonouno** [comment]
I wrote some comments. I can't check if these fixes are enough.

**Comment by unnonouno** [comment]
Please check all `CuDNN`.

**Comment by unnonouno** [comment]
How about using sphinxcontrib-spelling?
https://pypi.python.org/pypi/sphinxcontrib-spelling/

File: chainer/functions/array/broadcast.py (hunk 1 above)
Comment (okuta) [inline chainer/functions/array/broadcast.py]: I checked compiled document. It is OK.

File: chainer/functions/loss/negative_sampling.py [hunk 5]
```diff
@@ -170,17 +170,17 @@ def negative_sampling(x, t, W, sampler, sample_size):
 
     .. math::
 
-       f(x, p) = \\log \\sigma(x^\\top w_p) + \\
+       f(x, p) = \\log \\sigma(x^\\top w_p) +  \\\\
```
Comment (unnonouno) [inline chainer/functions/loss/negative_sampling.py]: Use two backslashes.

**Comment by unnonouno** [comment]
OK. LGTM

**Pull request merged**